tools/netlify-plugins/cypress-tests-runner/index.js

```diff
@@ -1,26 +1,58 @@
 import { spawnSync } from 'child_process';
+import { readFileSync, writeFileSync } from 'fs';
 
 export const onSuccess = ({ inputs }) => {
   const { nxRunCommand } = inputs;
   const deployURL = process.env.DEPLOY_PRIME_URL;
+  const buildId = process.env.BUILD_ID;
 
-  console.log(`Deployed site URL: ${deployURL}`);
+  updateCypressConfig(buildId, deployURL);
+  executeNxCommand(nxRunCommand);
+};
+
+/**
+ * To be able to run smoke tests and have nice results in PR comments
+ * we have to pass deployed site url and build id to cypress
+ *
+ * it is not possible to do so using env variables, so we have to
+ * modify nx config a bit to add/edit missing values
+ *
+ * @param {string} buildId
+ * @param {string} baseUrl
+ */
+function updateCypressConfig(buildId, baseUrl) {
+  console.log(`Deployed site URL: ${baseUrl}`);
+  console.log(`Deployed build id: ${buildId}`);
+
+  const pathToProjectConfig = '../../apps/storefront-e2e/project.json';
```

Comment:
We are hardcodeing path to the storefront `project.json`, but at the same time it looks like this script is a general one, how we will run it for PA or testfront?